src/main/scss/abstracts/_theme.scss

```diff
@@ -57,16 +57,9 @@ $semantics: (
   --focus-input-border: var(--accent-color);
   --focus-input-glow: color-mix(in sRGB, var(--accent-color) 15%, transparent);
 
-  // Deprecated
-  --focus-btn-primary: #{color.change(#0b6aa2, $alpha: 0.5)};
-  --focus-btn-secondary: #{color.change(#0b6aa2, $alpha: 0.5)};
-  --focus-btn-danger: #{color.change(#cc0003, $alpha: 0.5)};
-
   // State colors
-  --primary-hover: #0587d4;
-  --primary-active: #095683;
-  --danger-hover: #eb383b;
-  --danger-active: #b50003;
+  --primary-hover: var(--accent-color);
```

Comment:
Not really related to this - but updated these values to use the accent color rather than a custom blue.